plugins/connection/kubectl.py

```diff
@@ -65,6 +65,7 @@
       kubectl_extra_args:
         description:
           - Extra arguments to pass to the kubectl command line.
+          - Please be aware that this passes information directly on the command line and it could expose sensitive data.
```

Comment:
Should you add the `We recommend using the file based authentication options instead.` here too?

Reply:
Guess not? Looks like this was just merged.

Reply:
my bad, though I think the password/api key are the only sensitive auth bits you can pass in plain text that you need to work around with file based auth, the other kubectl options are either just paths to files or not sensitive AFAIK